contracts/extensions/MultisigPermissions.sol

```diff
@@ -508,7 +538,10 @@ contract MultisigPermissions is ColonyExtensionMeta, ColonyDataTypes, GetActionS
     bytes32 userPermissions = getUserRoles(msgSender(), _permissionDomainId);
     Motion storage motion = motions[_motionId];
 
-    if ((motion.requiredPermissions & ROOT_AND_ARCHITECTURE) == ROOT_AND_ARCHITECTURE) {
+    if (
+      getDominantSigs().includes(motion.sig) &&
```

Comment:
Can we not reference `dominantSigs` here?

Reply:
I'm not sure what you're asking? It's not declared in this scope, and we only need it the once here.